### PR TITLE
Reload reactive

### PR DESCRIPTION
**Pull request by PolinaAkimova**
Add for & for-block setup, rendering and changing

### CODE REVIEW

**Comment by PolinaAkimova** [comment]
if everything is good, I will add a description to Readme.md

**Comment by Destrokhen-main** [comment]
Yes, please add description in "readme".

I changed the branch. Everything is fine. i will fix some things

**Comment by Destrokhen-main** [comment]
I think I can get rid of sReactProperties. But not sure yet

File: index.js [hunk 1]
```diff
@@ -191,26 +260,53 @@ const sReact = (objectData) => {
...
         if (target[prop] !== n) {
+          target[prop] = n;
+          target.sReactProperty[prop].value = n;
+          
+          // вот чтоб вот это не делать можно теоретически в sReactProperty подставить type объектам
           const allTag = document.querySelectorAll(`*[tag='${prop}']`);
-
+          const forBlockTags = document.querySelectorAll(`[for-block="${prop}"]`);
+          const forElemsTags = document.querySelectorAll(`[for="${prop}"]`);
           const showTags = document.querySelectorAll(`*[show='${prop}']`);
```
Comment (Destrokhen-main) [inline index.js:286]: yes, it's true, but you can write array attribute `for` and you can write this object on `tag`


for `for` it's one logic, from `tag` it's another logic

**Pull request closed without merging**